### PR TITLE
#223: updates in the name of webpack@5 and webpack-cli@4 (also react@17)

### PR DESCRIPTION
**Pull request by espen42**
Mainly: react4xp lib and npm packages @2.0.0

Added cross-platform workaround for running `npm` command in windows.

### CODE REVIEW

File: src/main/resources/lib/nashornpolyfills.js [hunk 1]
```diff
@@ -0,0 +1,340 @@
+(() => {
```
Comment (espen42) [inline src/main/resources/lib/nashornpolyfills.js:1]: Wait what? Denne fila var uventet (trengs, men skal komme fra dependencies og ikke havne i src), her lurer jeg på om det har skjedd noe i bygginga.

Comment (espen42) [reply src/main/resources/lib/nashornpolyfills.js]: Ok, fjernet.

File: build.gradle [hunk 2]
```diff
@@ -1,7 +1,9 @@
+import org.apache.tools.ant.taskdefs.condition.Os;
+
 plugins {
     id 'com.enonic.defaults' version '2.0.1'
     id 'com.enonic.xp.app' version '3.0.0'
-    id "com.moowork.node" version "1.3.1"
+    id "com.github.node-gradle.node" version "3.1.0"
```
Comment (poi33) [inline build.gradle:6]: 👍

**Review by poi33 — APPROVED** [review]

**Comment by poi33** [comment]
Update readme with new info? @espen42

**Pull request merged**